docs/installation/installation-guide.md

```diff
@@ -6,28 +6,32 @@ sidebar_label: Installation guide
 
```

Comment:
General. Default Legend Studio config has duplicate Gitlab entries. It also disables creation of prototype projects.

Reply:
Done!